scripts/mkcloud

```diff
@@ -517,15 +540,15 @@ function ssh_password()
     SSH_ASKPASS=/root/echolinux
     cat > $SSH_ASKPASS <<EOSSHASK
 #!/bin/sh
-echo linux
+echo $admin_image_password
```

Comment:
That's not an option.
we want to use/test the same base image, we use later in productive setups, as well.
I think providing a way to change the root password, used for connecting to the admin image, is not such a big effort and beneficial also for others. Otherwise mkcloud always requires a "special" image.